pkg/api/admin/openshiftcluster.go

```diff
@@ -149,10 +184,11 @@ type NetworkProfile struct {
 	MTUSize      MTUSize      `json:"mtuSize,omitempty"`
 	OutboundType OutboundType `json:"outboundType,omitempty" mutable:"true"`
 
-	APIServerPrivateEndpointIP string           `json:"privateEndpointIp,omitempty"`
-	GatewayPrivateEndpointIP   string           `json:"gatewayPrivateEndpointIp,omitempty"`
-	GatewayPrivateLinkID       string           `json:"gatewayPrivateLinkId,omitempty"`
-	PreconfiguredNSG           PreconfiguredNSG `json:"preconfigureNSG,omitempty"`
+	APIServerPrivateEndpointIP string               `json:"privateEndpointIp,omitempty"`
+	GatewayPrivateEndpointIP   string               `json:"gatewayPrivateEndpointIp,omitempty"`
+	GatewayPrivateLinkID       string               `json:"gatewayPrivateLinkId,omitempty"`
+	PreconfiguredNSG           PreconfiguredNSG     `json:"preconfigureNSG,omitempty"`
+	LoadbalancerProfile        *LoadbalancerProfile `json:"loadBalancerProfile,omitempty"`
```

Comment:
super-nit: we are using `loadBalancerProfile` for this customer-facing option, but OutboundType uses the casing `Loadbalancer`. Might be good to align our casing for customer-facing options.

Reply:
I'm initially not for this change, but I'm open to changing it if the team wholly agrees.  The PRD states to use the aks user experience as a guide for this feature, so I'm trying to mirror [aks rest specs](https://learn.microsoft.com/en-us/rest/api/aks/managed-clusters/get?tabs=HTTP#containerservicenetworkprofile) convention and I'm hesitant to deviate from that.  Unfortunately the casing for `OutboundTypes` originate from the installer code therefore it was carried into the RP.